service/src/main/java/com/plhal/ares/service/DataServiceImpl.java

```diff
@@ -1,18 +1,22 @@
 package com.plhal.ares.service;
 
-import com.plhal.ares.model.Firma;
-import com.plhal.ares.model.DataRepository;
+import com.plhal.ares.dbLayer.Firma;
+import com.plhal.ares.dbLayer.DataRepository;
+import com.plhal.ares.dbLayer.FirmaRepository;
 
 /**
  * Concrete implementation for service layer, which contains business logic for communication with model.
  */
 
 public class DataServiceImpl implements DataService {
 
-    private final DataRepository dataRepository;
+    private DataRepository dataRepository;
 
-    public DataServiceImpl(DataRepository dataRepository) {
+    private FirmaRepository firmaRepository;
```

Comment:
Proc neni final?

Reply:
Tady mě zmátlo, že když jsem chtěl udělat DataRepositoryProperties jako final, tak to nebylo dobře. Ale tohle je teda jiný případ?

Reply:
Ted uplne presne nevim, co myslite. Odkazte me prosim na konkretni zpravu, abychom to vyjasnili.

Reply:
Vy myslite [toto](https://github.com/jiri-plhal/ares-api/pull/6#discussion_r584369654), ze? Ale tam neslo o to `final`. To tam melo zusatat a omlouvam se, nevsiml jsem si, ze jste ho odstranil. Slo o to, ze jste napsal nazev vlastnosti dane instance (nikoliv cele tridy - nebylo tam static) velkymi pismeny, jako by slo o konstantu (coz je v jave static final - viz muj odkaz).

Reply:
Ano tohle jsem myslel, tak to jsem Vas jen nepochopil. Uz je mi to jasnejsi. Opravim to.